src/NClient.Providers/NClient.Providers.Transport.SystemNetHttp/SystemNetHttpTransportRequestBuilder.cs

```diff
@@ -32,9 +32,6 @@ public Task<HttpRequestMessage> BuildAsync(IRequest request, CancellationToken c
                 RequestUri = uri
             };
 
-            if (request.Timeout.HasValue)
-                httpRequestMessage.SetTimeout(request.Timeout.Value);
```

Comment:
You should delete this extension, because now it is not used anywhere.

Reply:
Dropped